src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/Invitation.cs

```diff
@@ -135,6 +135,10 @@ public void RemoveAttachment(Attachment attachment)
             AddDomainEvent(new AttachmentRemovedEvent(Plant, Guid, attachment.FileName));
         }
 
+        public void AddParticipants(IReadOnlyCollection<Participant> participants) => _participants.AddRange(participants);
```

Comment:
Can't do! Should loop input participants, and call AddParticipant to execute business-logic there!
Same for AddCommPkgs and AddMcPkgs below

So, if you move these 3 methods to another derived class for Inivitation, AddParticipant should still be used (which again prohibir expose _participants as protected)

Reply:
Looping through 650000 participants... will check again which performance impact that has.